docker-compose-dev.yml

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+web:
+  build: .
+  command: /code/scripts/docker_devboot.sh master inveniomanage runserver -h 0.0.0.0 -p 5000 -d -r
+  ports:
+    - "5000:5000"
+  environment:
+    - CACHE_REDIS_HOST=cache
+    - CFG_DATABASE_HOST=db
+    - INVENIO_APP_CONFIG_ENVS=CACHE_REDIS_HOST,CFG_DATABASE_HOST
```

Comment:
Shall you add also `DEBUG=True` and `DEBUG_TB_INTERCEPT_REDIRECTS=False` to the DEV setup here?